src/main/java/com/keshavarzi/airplanestats/model/RouteEntity.java

```diff
@@ -0,0 +1,63 @@
+package com.keshavarzi.airplanestats.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "route", schema = "user_data")
+public class RouteEntity {
+
+    @Id
+    @Nonnull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", table = "route", unique = true, nullable = false)
+    private Long routeId;
```

Comment:
All variables should be made `final` if you don't explicitly expect them to be changed. Making them "immutable" will prevent bugs in future changes.

Reply:
If they are final, the need to be initialized and they can't change.  I can't make them final